sdk/eventhubs/azure_messaging_eventhubs_checkpointstore_blob/README.md

```diff
@@ -9,18 +10,64 @@ This crate provides a checkpoint store implementation for Azure Event Hubs using
 - **Concurrency Support**: Handle multiple processors with proper concurrency control
 - **Easy Integration**: Drop-in replacement for other checkpoint store implementations
 
-## Usage
+[Source code] | [Package (crates.io)] | [API reference documentation] | [Product documentation]
+
+## Getting started
+
+### Install the package(s)
+
+Install the Azure Event Hubs, Azure Blob Storage, and Blob Checkpoint Store client libraries for Rust with [Cargo]:
+
+```sh
+cargo add azure_messaging_eventhubs
+cargo add azure_storage_blob
+cargo add azure_messaging_eventhubs_checkpointstore_blob
+```
+
+### Prerequisites
+
+- A Rust Compiler. See [the rust compiler installation instructions](https://www.rust-lang.org/tools/install).
+- An [Azure subscription]
+- The [Azure CLI]
+- An [Event Hub namespace](https://learn.microsoft.com/azure/event-hubs/).
+- An Event Hub instance. You can create an Event Hub instance in your Event Hubs Namespace using the [Azure Portal](https://learn.microsoft.com/azure/event-hubs/event-hubs-create), or the [Azure CLI](https://learn.microsoft.com/azure/event-hubs/event-hubs-quickstart-cli).
+
+If you use the Azure CLI, replace `<your-resource-group-name>`, `<your-eventhubs-namespace-name>`, and `<your-eventhub-name>` with your own, unique names:
+
+Create an Event Hubs Namespace:
+
+```azurecli
+az eventhubs namespace create --resource-group <your-resource-group-name> --name <your-eventhubs-namespace-name> --sku Standard
+```
+
+Create an Event Hub Instance:
+
+```azurecli
+az eventhubs eventhub create --resource-group <your-resource-group-name> --namespace-name <your-eventhubs-namespace-name> --name <your-eventhub-name>
+```
+
+### Install dependencies
+
+Add the following crates to your project:
+
+```sh
+cargo add azure_identity tokio azure_messaging_eventhubs azure_storage_blob
+```
+
+### Authenticate the client
 
-Add this to your `Cargo.toml`:
+Before interacting with Azure services, you need a credential to use to authenticate and authorize the user.
 
-```toml
-[dependencies]
-azure_messaging_eventhubs_checkpointstore_blob = "0.1.0"
-azure_messaging_eventhubs = "0.20.0"
-azure_storage_blobs = "0.20.0"
-azure_identity = "0.20.0"
+The example shown below uses a [`DefaultAzureCredential`][default_cred_ref], which is appropriate for most local development environments. Additionally, we recommend using a managed identity for authentication in production environments. You can find more information on different ways of authenticating and their corresponding credential types in the [Azure Identity] documentation.
+
+The `DefaultAzureCredential` will automatically pick up on an Azure CLI authentication. Ensure you are logged in with the Azure CLI:
+
+```azurecli
+az login
```

Comment:
This is actually about to go away: #2873

Reply:
I know, but I until Charles' PR is in, this is what we have.